### ISSUE TITLE
Some standard cells with parallel series do not pass netgen LVS

### ISSUE THREAD

**Issue by d-m-bailey** [google/skywater-pdk#206]
CDL has parallel mos in series. layout has series mos in parallel. Logically equivalent, but not topologically equivalent.
https://skywater-pdk.slack.com/archives/C017UA7LEUV/p1604453653008500

## Expected Behavior
```
Circuit 1 contains 8 devices, Circuit 2 contains 8 devices.
Circuit 1 contains 11 nets,    Circuit 2 contains 11 nets.

Netlists match uniquely.

```

## Actual Behavior
```
Circuit 1 contains 8 devices, Circuit 2 contains 10 devices. *** MISMATCH ***
Circuit 1 contains 11 nets,    Circuit 2 contains 12 nets. *** MISMATCH ***

```

## Steps to Reproduce the Problem

1. extract the mag view 
1. run ext2spice
1. compare to cdl

## Specifications

- Version: commit 5cd70ed
- Platform: MacOS

Sample layout will follow

**Comment by mithro** [comment]
@d-m-bailey - I think you closed this because it is a duplicate of https://github.com/google/skywater-pdk/issues/205 ?
